src/tests/ftest/control/dmg_telemetry_io_latency.py

```diff
@@ -339,14 +340,132 @@ def test_io_latency_telmetry_metrics(self):
         if errors:
             self.fail("Test FAILED")
 
+    def verify_io_dtx_committed_metrics(self, metrics_data, test_metrics, transfer_size):
+        """Verify IO dtx metrics after running IOR.
+
+        Args:
+            metrics_data (dict): dictionary of io dtx "committed" metrics
+            test_metrics (dict): io dtx "committed" telemetry metrics
+            transfer_size(str): transfer_size used with ior
+
+        Returns:
+            status: (bool) True if metrics are verified for transfer size
+
+        """
+        block_size = self.params.get("block_size", "/run/*")
+        repetitions = self.params.get("repetitions", "/run/*")
+        test_ops = 3
+        status = True
+
+        metrics = {}
+        for test_metric in test_metrics:
+            metrics[test_metric] = 0
+            for host in self.hostlist_servers:
+                # test assumes one engine per host
+                for rank in self.server_managers[-1].get_host_ranks([host]):
+                    for target in range(self.server_managers[-1].get_config_value("targets")):
+                        value = metrics_data[test_metric][host][str(rank)][str(target)]["-"]
+                        metrics[test_metric] = metrics[test_metric] + value
+        dtx_value = metrics["engine_io_dtx_committed"]
+        min_value = metrics["engine_io_dtx_committed_min"]
+        max_value = metrics["engine_io_dtx_committed_max"]
+        mean_value = metrics["engine_io_dtx_committed_mean"]
+        stddev_value = metrics["engine_io_dtx_committed_stddev"]
+
+        # DTX committed value should be equal to the number of transactions.
+        if (transfer_size == "512"):
```

Comment:
(pylint-superfluous-parens) Unnecessary parens after 'if' keyword

Reply:
This class of error would be really easy to fix, it would be great if you could do it when working on the PR to avoid someone else having to tidy it up later.  Please fix if you need to push again.